build/deploy/schema-manager.libsonnet

```diff
@@ -48,5 +65,32 @@ local rid_schema_mount = {
         },
       },
     } else null,
+    SCDSchemaManager: if metadata.cockroach.shouldInit && metadata.enableScd then base.Job(metadata, 'scd-schema-manager') {
+      spec+: {
+        template+: {
+          spec+: {
+            volumes_: {
+              client_certs: volumes.volumes.client_certs,
+              ca_certs: volumes.volumes.ca_certs,
+              scd_schema: scd_schema_vol,
+            },
+            soloContainer:: base.Container('scd-schema-manager') {
+              image: metadata.schema_manager.image,
+              args_:: {
+                cockroach_host: 'cockroachdb-balanced.' + metadata.namespace,
+                cockroach_port: metadata.cockroach.grpc_port,
+                cockroach_ssl_mode: 'verify-full',
+                cockroach_user: 'root',
+                cockroach_ssl_dir: '/cockroach/cockroach-certs',
+                db_version: metadata.schema_manager.desired_scd_db_version,
+                schemas_dir: scd_schema_mount.mountPath,
+
```

Comment:
nit: extra blank line

Reply:
I'm following [general best practices](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) I think.